### PR TITLE
Typos and desc style

### PR DESCRIPTION
**Pull request by greenteaguzzler**
## About The Pull Request

- Fixes some typos
- Makes descriptions consistent, "It's a/an...' changed to "A/an..."

## Why It's Good For The Game

Less weird syntax

## Changelog
:cl:
spellcheck: fixed a few typos
spellcheck: made descriptions more consistent
/:cl:

### CODE REVIEW

File: code/modules/hivemind/machines.dm [hunk 1]
```diff
@@ -56,7 +56,7 @@
 	else if (health < max_health * 0.75)
 		to_chat(user, SPAN_WARNING("It has numerous dents and deep scratches."))
 	else if (health < max_health)
-		to_chat(user, SPAN_WARNING("It's a bit scratched and has dents."))
+		to_chat(user, SPAN_WARNING("A bit scratched and has dents."))
```
Comment (TheShown911) [inline code/modules/hivemind/machines.dm:59]: This should stay as It's as it's a general message to the chat, not a description.

File: code/modules/holodeck/HolodeckObjects.dm [hunk 2]
```diff
@@ -113,11 +113,11 @@
 		return
 
 	if(istype(W, /obj/item/tool/screwdriver))
-		to_chat(user, (SPAN_NOTICE("It's a holowindow, you can't unfasten it!")))
+		to_chat(user, (SPAN_NOTICE("A holowindow, you can't unfasten it!")))
```
Comment (TheShown911) [inline code/modules/holodeck/HolodeckObjects.dm:116]: +1

File: code/modules/holodeck/HolodeckObjects.dm [hunk 3]
```diff
@@ -113,11 +113,11 @@
...
 	else if(istype(W, /obj/item/tool/crowbar) && reinf && state <= 1)
-		to_chat(user, (SPAN_NOTICE("It's a holowindow, you can't pry it!")))
+		to_chat(user, (SPAN_NOTICE("A holowindow, you can't pry it!")))
```
Comment (TheShown911) [inline code/modules/holodeck/HolodeckObjects.dm:118]: +1

File: code/modules/holodeck/HolodeckObjects.dm [hunk 4]
```diff
@@ -113,11 +113,11 @@
...
 	else if(istype(W, /obj/item/tool/wrench) && !anchored && (!state || !reinf))
-		to_chat(user, (SPAN_NOTICE("It's a holowindow, you can't dismantle it!")))
+		to_chat(user, (SPAN_NOTICE("A holowindow, you can't dismantle it!")))
```
Comment (TheShown911) [inline code/modules/holodeck/HolodeckObjects.dm:120]: +1

File: code/modules/holodeck/HolodeckObjects.dm [hunk 5]
```diff
@@ -183,7 +183,7 @@
 
 /obj/structure/bed/chair/holochair/attackby(obj/item/W as obj, mob/user as mob)
 	if(istype(W, /obj/item/tool/wrench))
-		to_chat(user, (SPAN_NOTICE("It's a holochair, you can't dismantle it!")))
+		to_chat(user, (SPAN_NOTICE("A holochair, you can't dismantle it!")))
```
Comment (TheShown911) [inline code/modules/holodeck/HolodeckObjects.dm:186]: +1

File: code/modules/mob/living/silicon/robot/drone/drone_items.dm [hunk 6]
```diff
@@ -36,7 +36,7 @@
 
 	for(var/mob/M in T)
 		if(istype(M,/mob/living/simple_animal/lizard) || ismouse(M))
-			src.loc.visible_message(SPAN_DANGER("[src.loc] sucks [M] into its decompiler. There's a horrible crunching noise."),SPAN_DANGER("It's a bit of a struggle, but you manage to suck [M] into your decompiler. It makes a series of visceral crunching noises."))
+			src.loc.visible_message(SPAN_DANGER("[src.loc] sucks [M] into its decompiler. There's a horrible crunching noise."),SPAN_DANGER("A bit of a struggle, but you manage to suck [M] into your decompiler. It makes a series of visceral crunching noises."))
```
Comment (TheShown911) [inline code/modules/mob/living/silicon/robot/drone/drone_items.dm:39]: +1

File: code/modules/power/lighting.dm [hunk 7]
```diff
@@ -35,7 +35,7 @@
 
 	switch(src.stage)
 		if(1)
-			to_chat(user, "It's an empty frame.")
+			to_chat(user, "An empty frame.")
```
Comment (TheShown911) [inline code/modules/power/lighting.dm:38]: +1

**Review by TheShown911 — COMMENTED** [review]
Soulless

**Comment by hyperioo** [comment]
You're doing god's work, never realized there were that many "It's a"

**Pull request merged**